aospy/calc.py

```diff
@@ -8,14 +8,13 @@
 
 import numpy as np
 import xray
-
```

Comment:
I put builtin imports first, followed by a space, followed by other package imports, followed by a space, followed by relative imports within the package.  So this space should remain.